deploy/kubernetes/base/csi-driver-instance.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: storage.k8s.io/v1beta1
+apiVersion: storage.k8s.io/v1
```

Comment:
v1beta1 is not supported on modern Kind clusters.

Reply:
This API was removed in 1.27 and the v1 API has been available since 1.24 so I think this change is ok to make.